examples/patterns/contextual-menu/contextual-menu.html

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Contextual menu
+category: _patterns
+---
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Modi natus <span class="p-contextual-menu">
+    <a href="#" class="u-toggle" aria-controls="#menu" aria-expanded="false" aria-haspopup="true">Link</a>
```

Comment:
`u-toggle`should be part of this pattern. So you can use this pattern with out importing utilities.